docs/governance.rst

```diff
@@ -0,0 +1,129 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
+maintainer of ``mybinder.org``, a public service and demonstration of
+the BinderHub technology.
+
+Membership
+~~~~~~~~~~
+
+Membership in the Binder Project is open to anyone who is nominated by
+an existing member. Members form the trusted group of individuals who
+manage the Binder Project.
+
+Membership comes with `privileges`_ and `responsibilities`_.
+
+Currently, the Binder Project team overlaps 100% with `the BinderHub
+team`_. However, these two groups may diverge in their membership over
+time.
+
+The authoritative list of members and their roles (alphabetical sorting):
+* Tim Head (red)
+* Min Ragan-Kelley (team leader)
+* Yuvi Panda (red)
+* Erik Sundell (blue)
+* Carol Willing (red)
+* Your Name (red or blue)
+
+Team privileges
```

Comment:
These apply to both red+blue team members unless otherwise stated, right?

Reply:
More broadly I'm just not sure where these roles came from?  I don't remember discussing this at the last meeting?

Reply:
These came from discussions in #42. There was no in person discussions.